### PR TITLE
fix(Navigation): Updated navbar padding & underline styling so naviga…

### PR DESCRIPTION
**Pull request by cseitz**
…tion button text is vertically centered in navigation bar

### CODE REVIEW

File: src/components/navigation/Navbar.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default {
         scaleFactor = 0.5;
       }
       let n = (30 * scaleFactor) * (1 - this.progress) + (10 * scaleFactor) * this.progress
-      return `${n}px 10px ${10 + n}px 10px`;
+      return `${n}px 10px ${n}px 10px`;
```
Comment (cseitz) [inline src/components/navigation/Navbar.vue:35]: We want uniform padding

File: src/components/navigation/Navbar.vue [hunk 2]
```diff
@@ -113,7 +113,7 @@ export default {
 .navbar {
   position: fixed;
   width: 100vw;
-  height: 55px;
+  // height: 55px;
```
Comment (cseitz) [inline src/components/navigation/Navbar.vue:116]: Explicit height is problematic. Height will grow to fit buttons. Control button sizing or padding to control height of navbar.

File: src/scss/underline.scss [hunk 3]
```diff
@@ -12,7 +12,7 @@
     padding-right: 0px;
     //background: none repeat scroll 0 0 transparent;
     background: $color;
-    margin-top: 5px;
+    transform: translateY(5px);
```
Comment (cseitz) [inline src/scss/underline.scss:15]: margin-top was causing items w/ underline functionality to have 2px of extra spacing on the bottom, causing vertical alignment issues. this transform implements the same relative offset for the underline without growing the element by 2px

**Comment by cseitz** [comment]
@toBeOfUse lmk what you think

**Comment by toBeOfUse** [comment]
Thanks, this is much better. The drop shadow underneath the navbar is subtle enough in color on my monitor that I didn't notice that the text isn't vertically centered.

**Pull request merged**